recipes/itk/all/conanfile.py

```diff
@@ -0,0 +1,241 @@
+        cmake.definitions["Module_ITKLevelSets"] = "ON"
+        cmake.definitions["Module_ITKLevelSetsv4"] = "ON"
+
+        cmake.definitions["Module_ITKMarkovRandomFieldsClassifiers"] = "ON"
+        cmake.definitions["Module_ITKMathematicalMorphology"] = "ON"
+        cmake.definitions["Module_ITKMetricsv4"] = "ON"
+        cmake.definitions["Module_ITKNarrowBand"] = "ON"
+        cmake.definitions["Module_ITKNeuralNetworks"] = "ON"
+        cmake.definitions["Module_ITKOptimizers"] = "ON"
+        cmake.definitions["Module_ITKOptimizersv4"] = "ON"
+        cmake.definitions["Module_ITKPDEDeformableRegistration"] = "ON"
+        cmake.definitions["Module_ITKPath"] = "ON"
+        cmake.definitions["Module_ITKPolynomials"] = "ON"
+        cmake.definitions["Module_ITKQuadEdgeMeshFiltering"] = "ON"
+        cmake.definitions["Module_ITKRegionGrowing"] = "ON"
+        cmake.definitions["Module_ITKRegistrationCommon"] = "ON"
+        cmake.definitions["Module_ITKRegistrationMethodsv4"] = "ON"
+        cmake.definitions["Module_ITKReview"] = "ON"
+        cmake.definitions["Module_ITKSignedDistanceFunction"] = "ON"
+        cmake.definitions["Module_ITKSmoothing"] = "ON"
+        cmake.definitions["Module_ITKSpatialFunction"] = "ON"
+        cmake.definitions["Module_ITKThresholding"] = "ON"
+
+        cmake.definitions["Module_ITKVideoCore"] = "OFF"
+        cmake.definitions["Module_ITKVideoFiltering"] = "OFF"
+        cmake.definitions["Module_ITKVideoIO"] = "OFF"
+        cmake.definitions["Module_ITKVoronoi"] = "ON"
+        cmake.definitions["Module_ITKVTK"] = "OFF"
+        cmake.definitions["Module_ITKWatersheds"] = "ON"
+        cmake.definitions["Module_ITKDICOMParser"] = "OFF"
+
+        # Disabled on Linux (link errors)
+        cmake.definitions["Module_ITKLevelSetsv4Visualization"] = "OFF"
+        cmake.definitions["Module_ITKVtkGlue"] = "OFF"
+
+        # Disabled because Vxl vidl is not build anymore
+        cmake.definitions["Module_ITKVideoBridgeVXL"] = "OFF"
+
+        if not tools.os_info.is_windows:
+            cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = "ON"
+        cmake.configure(build_folder=self.build_subfolder)
+        cmake.build()
+        cmake.install()
+
+    def cmake_fix_macos_sdk_path(self, file_path):
+        # Read in the file
+        with open(file_path, 'r') as file:
+            file_data = file.read()
+
+        if file_data:
+            # Replace the target string
+            file_data = re.sub(
+                # Match sdk path
+                r';/Applications/Xcode\.app/Contents/Developer/Platforms/MacOSX\.platform/Developer/SDKs/MacOSX\d\d\.\d\d\.sdk/usr/include',
+                '',
+                file_data,
+                re.M
+            )
+
+            # Write the file out again
+            with open(file_path, 'w') as file:
+                file.write(file_data)
+
+    def cmake_fix_path(self, file_path, package_name):
```

Comment:
You can probably use the experimental [`patch_config_paths()`](https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#patch-config-paths-experimental) to achieve the same thing.